app/views/components/_topic_list.html.erb

```diff
@@ -15,13 +16,27 @@
 %>
 <% if items.any? %>
   <%= tag.ul(class: ul_classes, lang: "en") do %>
-    <% items.each do |item| %>
+    <% items.each_with_index do |item, index| %>
       <li class="app-c-topic-list__item">
         <%=
           link_to(
             item[:text],
             item[:path],
-            data: item[:data_attributes],
+            data: {
+              **item[:data_attributes] || {},
+              ga4_link: {
```

Comment:
Some of these data attributes are here regardless of whether any GA4 data attributes are passed. Is there a situation where they're not passed? Should we make them only appear if `ga4_data` exists? If this data is always here, why not hard code the link tracker module to the parent?